documentation/breaking-changes/upcoming-breaking-changes/Az.StorageSync-UpcomingBreakingChanges.md

```diff
@@ -2,27 +2,17 @@
 
  The following cmdlets were affected this release:
 
-
-
-
-### **Invoke-AzStorageSyncFileRecall**
```

Comment:
No need to update this file. Please revert it.

Reply:
Reverted it.